lms/static/js/staff_debug_actions.js

```diff
@@ -17,95 +19,21 @@ var StaffDebug = (function(){
         uname =  $('#sd_fu_' + locname).attr('placeholder');
     }
     return uname;
-  }
-
-  do_idash_action = function(action){
```

Comment:
@dan-f We removed all three ```staff_debug_info``` links and made a single link that's why I removed these functions and added a single link functionality here.